go.mod

```diff
@@ -121,6 +121,7 @@ require (
 	github.com/go-sql-driver/mysql v1.7.1 // indirect
 	github.com/go-stack/stack v1.8.1 // indirect
 	github.com/go-viper/mapstructure/v2 v2.0.0-alpha.1 // indirect
+	github.com/gofrs/uuid/v5 v5.0.0 // indirect
```

Comment:
we're using only typeid which uses UUIDv7, i think ULID is used for the integrations for connection tokens. @daabr am i correct?

Reply:
Integrations use "shortuuid" to generate tokens. ULID was suggested previously as a replacement for the general usage of UUIDv4, but we decided to use TypeID (over UUIDv7) instead.

BTW, integrations will continue to use "shortuuid" because they need 100% random values which are **not** K-sortable by time.